react/features/base/config/functions.any.ts

```diff
@@ -89,7 +89,47 @@ export function getFeatureFlag(state: IReduxState, featureFlag: string) {
  * @returns {boolean}
  */
 export function getDisableRemoveRaisedHandOnFocus(state: IReduxState) {
-    return state['features/base/config']?.disableRemoveRaisedHandOnFocus || false;
+    return state['features/base/config']?.raisedHands?.disableRemoveRaisedHandOnFocus || false;
+}
+
+/**
+ * Selector used to get the disableLowerAllHand.
+ *
+ * @param {Object} state - The global state.
+ * @returns {boolean}
+ */
+export function getDisableLowerAllHand(state: IReduxState) {
```

Comment:
I think we should use a single option for disabling the lower hand functionality. I'm not sure who would want to allow one but not the other...